common/build.gradle

```diff
@@ -138,6 +139,11 @@ dependencies {
     implementation "com.google.code.gson:gson:$rootProject.ext.gsonVersion"
     implementation "androidx.browser:browser:$rootProject.ext.browserVersion"
     implementation "androidx.constraintlayout:constraintlayout:$rootProject.ext.constraintLayoutVersion"
+    // Bouncycastle/spongycastle
```

Comment:
Do we need these?

I recently got a question from @titorodi and @AndiMiko as to whether or not we can remove these dependencies in broker...

Reply:
We're currently using them for certificate interface, and I pulled this into common for SP800 key gen,.

Reply:
I stole Richard's implementation from MSAL-cpp.  We can drop it.